packages/manager/modules/navbar/src/notifications-menu/constants.js

```diff
@@ -1,14 +1,17 @@
 export const ACKNOWLEDGED_STATUS = ['acknowledged', 'completed', 'unknown'];
 
-export const ACTIVE_STATUS = ['acknowledged', 'delivered'];
+export const ACTIVE_STATUS = 'delivered';
 
-export const ANIMATED_STATUS = ['incident', 'error', 'warning'];
```

Comment:
I'm not sure the animation feature should be removed

Reply:
as per functional doc, bell icon is animated if there is a new notification irrespective of the type of notification.